vowpalwabbit/core/src/reductions/scorer.cc

```diff
@@ -21,6 +22,11 @@ using namespace VW::config;
 
 namespace
 {
+struct options_scorer
```

Comment:
naming convention from the PR I have out would be `options_scorer_v1`, with the idea that new versions could be introduced later. Open to other convensions, thoughts @lokitoth ?